tests/pds/dataservice_test.go

```diff
@@ -293,6 +293,100 @@ var _ = Describe("{ScaleUPDataServices}", func() {
 	})
 })
 
+var _ = Describe("{RunTpccWorkloadOnDataServices}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RunTpccWorkloadOnDataServices", "Runs TPC-C Workload on Postgres and MySQL Deployment", nil, 0)
+	})
+
+	It("Deploy , Validate and start TPCC Workload", func() {
+		for _, ds := range params.DataServiceToTest {
+			if ds.Name == "PostgreSQL" {
+				log.InfoD("Deploying, Validating and Running TPCC Workload on %v Data Service ", ds.Name)
+				deployAndTriggerTpcc(ds.Name, ds.Version, ds.Image, ds.Version, ds.Image, int32(ds.Replicas))
+			}
+			if ds.Name == "MySQL" {
+				log.InfoD("Deploying, Validating and Running TPCC Workload on %v Data Service ", ds.Name)
+				deployAndTriggerTpcc(ds.Name, ds.Version, ds.Image, ds.Version, ds.Image, int32(ds.Replicas))
+			}
+		}
+	})
+	JustAfterEach(func() {
+		defer func() {
+			if !isDeploymentsDeleted {
+				Step("Delete created deployments")
+				resp, err := pdslib.DeleteDeployment(deployment.GetId())
+				Expect(err).NotTo(HaveOccurred())
+				Expect(resp.StatusCode).Should(BeEquivalentTo(http.StatusAccepted))
+			}
+		}()
+
+		defer EndTorpedoTest()
```

Comment:
make this a fist statement

Reply:
Hi Leela, can you pls elaborate on this ask

Reply:
make the defer EndTorpedoTest() as the first line in JustAfterEach, so that it gets executed even if there is error in DeleteDeployment call

Reply:
Done